docs/API/InstrumentFormat.md

```diff
@@ -370,6 +488,30 @@ input. It has the following form:
 
 `Options`: None
 
+### 2.2.3: RadioLabelElement
+
+To be used above one or more RadioElements of the same cardinality. The labels are
+aligned above the radio inputs.
```

Comment:
I don't see why a special element type is needed for this and not just a normal label and the element group/table types. It seems to just add extra maintenance overhead, but this can be addressed later.

Reply:
It is a bit awkward I agree.